### ISSUE TITLE
Cartopy Ocean Feature 10m Very Slow to create Movie

### ISSUE THREAD

**Issue by neocronomican** [SciTools/cartopy#1051]
Hello, 

I am using cartopy in order to create countourf animation. 

I have problem that when I use NaturalEarthFeature OCEAN with 10 m resolution, the ploltting process is very slow if I compare with 50m resolution. I have install fiona in my system  but it is not help at all..

Please any suggestion to solve this issue.

Thank you in advance

**Comment by greglucas** [comment]
I ran into a similar issue and tried to fix it, have you tried utilizing my pull request? #1000 

There were two issues with the 10 m resolutions that I fixed that you can look at the details within that pull request, or pull from my branch to directly use the updates. I'm not sure when these will be merged into master, but I hope this helps for now!

**Comment by neocronomican** [comment]
Dear Mr Greglucas,

Before make this thread I already try your patch but performance just the same. 

So I try search where the problem, and I found the problem is in my Fiona package. I have install Fiona by using conda and the instalation success, but I dont why I failed when I try to import fiona. So I remove fiona and gdal conda package then re-install use by using pip. Finally I can import the fiona and retry your patch again.

The result is I confirm that your patch (crs.py and shapereader.py) has improve performance for use OCEAN 10 m resolution. Thank you very much for your effort.

Best regards
